.storybook/webpack.config.js

```diff
@@ -0,0 +1,19 @@
+"use strict";
+
+const path = require("path");
+
+module.exports = {
+  resolve: {
+    alias: {
+      "devtools": "ff-devtools-libs",
+      "sdk": "ff-devtools-libs/sdk"
+    },
+    extensions: ["", ".js", ".jsm"],
+    root: path.join(__dirname, "node_modules")
+  },
+  module: {
+    loaders: [
+      { test: /\.css$/, loader: "style-loader!css-loader" }
+    ]
+  }
+};
```

Comment:
this is mostly a placeholder, I think in reality we can re-use much of the existing webpack.config.js